### PR TITLE
Install Java 7 on agent for mgmt ci builds

### PR DESCRIPTION
**Pull request by benbp**
Java 7 was removed from the hosted agents on February 8th 2021, via https://github.com/actions/virtual-environments/issues/2446, which has the effect of breaking all mgmt ci builds. This should install Java 7 where it's still used.

### CODE REVIEW

**Comment by benbp** [comment]
All my builds are getting scheduled on the old agents... :(

**Review by joshfree — APPROVED** [review]

**Review by weidongxu-microsoft — APPROVED** [review]

File: eng/pipelines/templates/stages/archetype-sdk-management.yml [hunk 1]
```diff
@@ -29,14 +29,21 @@ stages:
...
       - task: Maven@3
         displayName: 'Build'
         inputs:
           mavenPomFile: sdk/${{parameters.ServiceDirectory}}/pom.mgmt.xml
           goals: 'compile'
           options: '$(BuildOptions) $(ProfileFlag) "-DpackageOutputDirectory=$(Build.ArtifactStagingDirectory)" -DskipTests'
           mavenOptions: '$(MemoryOptions) $(LoggingOptions)'
-          javaHomeOption: 'JDKVersion'
+          javaHomeOption: path
+          jdkDirectory: '/usr/lib/jvm/zulu-7-azure-amd64'
```
Comment (weshaggard) [inline eng/pipelines/templates/stages/archetype-sdk-management.yml:46]: Won't this break for the java 1.8 builds?

Comment (weidongxu-microsoft) [reply eng/pipelines/templates/stages/archetype-sdk-management.yml:46]: Ah, possible. Might need a condition as well? (https://docs.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#conditionally-set-a-task-input)

**Comment by benbp** [comment]
Closing for now until a decision is made re: the java 7 build approach.

**Pull request closed without merging**